packages/customWidgets/calendar-web/src/components/CalendarContainer.ts

```diff
@@ -430,17 +432,26 @@ export default class CalendarContainer extends Component<Container.CalendarConta
     };
 
     private handleOnClickEvent = (eventInfo: Container.EventInfo) => {
-        mx.data.get({
-            guid: eventInfo.guid,
-            callback: this.executeEventAction,
-            error: error => window.mx.ui.error(`Error while executing action: ${error.message}`)
+        if (this.state.isExecuting && this.props.disabledDuringAction) {
+            return;
+        }
+
+        this.setState({ isExecuting: true }, () => {
+            mx.data.get({
+                guid: eventInfo.guid,
+                callback: this.executeEventAction,
+                error: error => window.mx.ui.error(`Error while executing action: ${error.message}`)
```

Comment:
The error should also reset the isExecuting state.

Reply:
fixed